homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -48,6 +49,10 @@
 
 SCAN_INTERVAL = timedelta(minutes=5)
 
+PARALLEL_UPDATES = 1
+
+MS_BETWEEN_API_CALLS = 0.5
```

Comment:
The PRs subject seems to indicate 0.5 _second_, while this seems to represent 0.5 _millisecond_. Is this value right?

Reply:
`asyncio.sleep` takes seconds as a parameter.

https://docs.python.org/3/library/asyncio-task.html#asyncio.sleep

The constant name could be clarified, but there's nothing wrong in the logic.